pkg/types/azure/doc.go

```diff
@@ -0,0 +1,7 @@
+package azure
+
+// Package azure contains Azure-specific structures for installer
```

Comment:
These seem like they are in the wrong place.

Reply:
Can you elaborate?